### DISCUSSION TITLE
lotus daemon can not sync message

### DISCUSSION THREAD

**Discussion by ZhuJerry12345** [filecoin-project/lotus#11497] [Build and Sync]
I install lotus & lily and init them with & without a snapshot on two computers. This problem has occurred in all the 8 situations : I start a daemon, after it finished head sync, it failed to sync message as follow:
lotus daemon:
![image](https://github.com/filecoin-project/lotus/assets/76514614/f907ecce-c233-40ba-a5aa-c1ad7bd70f07)
lotus sync status:
![image](https://github.com/filecoin-project/lotus/assets/76514614/b002c7a8-31a4-426c-880f-aba8a98b9856)

I guess it could be due to the following reasons ?
1. no public ip. 
both the  computers have no public ip, and i can't access to the router to forwarding ports

2. gfw. 
I'm in china and i have tried to use a proxy on one computer, which succeeds to ping & curl google but doesn't help with the problem. What's more, i don't know whether the net manager has maken additional restrictions to blockchains  beyond the gfw.
Due to some reason, it's difficult to use a proxy on the another one.

Infomation:
lotus version
Daemon:  1.25.0+mainnet+git.7e6b20cd9+api1.5.0
Local: lotus version 1.25.0+mainnet+git.7e6b20cd9

machine configure:
one:
ubuntu 22.04,  Intel 11700k, no GPU, 128GB Ram,  12TB HDD
another:
ubuntu SMP, AMD EPYC 7302, 2060s, 1TB Ram, 12TB HDD

lotus net peers:
12D3KooWSZSTpBy78SWYjgC2nz8ZT4zKonRN1FBbin1gdbMccdAx, [/ip4/208.70.27.174/tcp/38231]
12D3KooWSN4qPNbkiWQupiEmRVDFxTP7Ybecb2JZ6Jpw9oNk5hKL, [/ip4/128.136.157.164/tcp/8089]
12D3KooWRhP1G62pkJVfnvUguGyfBpwRiP8XWk8AEp6kSkFuGSZv, [/ip4/113.29.246.219/tcp/18376]
12D3KooWRfDyn63dCp6shTA24bZ27vg72T28rCLvGzgwh1E8SMLn, [/ip4/154.42.3.178/tcp/24001]
etc 

lotus net reachability:
AutoNAT status:  Private

**Comment by TippyFlitsUK** [comment]
Are you able to ping any of the peers? eg `ping 154.42.3.178`

**Reply by ZhuJerry12345** [reply]
yes
![image](https://github.com/filecoin-project/lotus/assets/76514614/5a09ee28-225e-412f-8d76-ae9afe39c449)

**Reply by TippyFlitsUK** [reply]
Are you able,e to also test with a VPN?

**Reply by ZhuJerry12345** [reply]
oh , i read some other issues and it seems that "dorequest failed for all peers" is a commen case as all the peers i connect have no corresponding content. But it's very Disappointing that it only sync 8000 blocks in 2 days, which means i need 2 years to sync the full chain data. I remember that 2 years ago the officia had offered the full chain snapshot, Why don't they do it now？ Is there anyway i can sync the chain data more fast?
